firebase-inappmessaging/firebase-inappmessaging.gradle

```diff
@@ -116,17 +116,17 @@ dependencies {
     implementation project(':protolite-well-known-types')
     implementation project(':transport:transport-api')
     implementation project(':firebase-datatransport')
+    implementation project(':firebase-installations-interop')
```

Comment:
Wondering if you should depend on the released version vs the HEAD? Idk whats the standard way in firebase.